example/macos/Runner.xcodeproj/project.pbxproj

```diff
@@ -148,6 +150,15 @@
 			name = Frameworks;
 			sourceTree = "<group>";
 		};
+		D76E93BA226BCA8F00F7932B /* Recovered References */ = {
```

Comment:
The configs should be referenced in the project, rather than being considered recovered files. Please rename this group in the file tree "Configs", move it up between Resources and Frameworks, and (for ease of inspection) add Generated.xcconfig to that group as well (make sure to uncheck the option to add it to any of the targets)

Reply:
We don't reference the Generated.xcconfig in our iOS projects, should this be changed in the templates for those as well?

Reply:
It doesn't affect the build in any way, it just means that if you have the project open you can easily inspect all the configs. I find it helpful in general, but if nobody has ever complained it's up to you whether it's worth changing iOS.